webapp/src/widgets/Cards/Cards.js

```diff
@@ -1,20 +1,53 @@
 import React from 'react'
-import { Card } from 'react-bootstrap'
+import { Card, Button } from 'react-bootstrap'
 import classNames from 'classnames'
 import './Cards.css'
+import { USERITEM_MODAL_TYPES } from '../../features/UserItemModals/UserItemModals.constants'
+import { useDispatch, useSelector } from 'react-redux'
+import { setSelectedUserItemModal } from '../../features/UserItem/userItem.slice'
+import { selectedUserItemModal } from '../../features/UserItem/userItem.selectors'
```

Comment:
use `select` as prefix in selectors not `selected. so, selectUserItemModal

Reply:
updated